microsite/src/main/tut/typeclass/Unfoldable1.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: docs
+title:  "Unfoldable1"
+---
+
+# Unfoldable1 [![GitHub](../img/github.png)](https://github.com/scalaz/scalaz/blob/series/8.0.x/base/shared/src/main/scala/scalaz/tc/unfoldable1.scala)
+
+* `Unfoldable1` identifies non-empty data structures which can be unfolded.*
```

Comment:
I think this is *very* unclear and can be improved on, but it's fine to me to leave it for this PR.

Reply:
Something to the effect that `Unfoldable1` is analogous to `Foldable1`, which corresponds to the anamorphisms of coalgebraic structures would be nice.